### DISCUSSION TITLE
Unable do download Immich models

### DISCUSSION THREAD

**Discussion by luucasv** [immich-app/immich#14795] [Q&A] [answered]
I'm unable to download Immich models. I see people with the same issue in the past, but I'm unsure what the current solution is.
Here are some sample logs:

Downloading visual model 'ViT-B-32__openai'. This may take a while.
Downloading detection model 'buffalo_l'. This may take a while.
Failed to load visual model 'ViT-B-32__openai'.  Clearing cache.                                    
Attempted to clear cache for model 'ViT-B-32__openai', but cache directory does not exist                                              
Failed to load detection model 'buffalo_l'. Clearing cache.                                    
Downloading visual model 'ViT-B-32__openai'. This may take a while.                                  
...

**Comment by luucasv** [comment, marked as answer]
I had found before that you could manually download the models and add them to the volume folder. After clearing my mind I was able to find [this issue](https://github.com/immich-app/immich/issues/6616) where some kindly share their folder structure for the volume.

**Comment by luucasv** [comment]
My question now is, is this going to be a permanent solution for me? Why wasn't immich able to download the files itself?

**Reply by tye-mustafa** [reply]
its normal just download it yourself and it will load it from it,
